### ISSUE TITLE
When in the decryption and copying files section the process can hang indefinitely

### ISSUE THREAD

**Issue by fac7orx** [13xforever/ps3-disc-dumper#6]
When decrypting a game such a killzone 3 the "decrypting and copying files..." process hangs at around 10.88GB transferred, and cannot recover. The transfer then continues to bounce around this number and never go beyond it.

Using a compatible drive on the list: WH14NS40

https://rpcs3.net/quickstart

**Comment by 13xforever** [comment]
Attach log. But usually this means your disc is scratched, so it has to retry bad sectors multiple times, which is super slow.

**Comment by fac7orx** [comment]
Hi you can get the log file here:

https://www.mediafire.com/file/7t5rt9gzbn897vn/2019-09-06.log/file

There may be remnants of me attempting another run. I did do another run
and so far is is on 12:13GB so it got passed it for some reason. However, I
am fearful  it may hang at some other spot. Will report back if it does.

**Comment by fac7orx** [comment]
Hi,

It seems to have hanged again at 24.38GB this time. Here is the new log file:

http://www.mediafire.com/file/5i4w6pysgyvwawp/2019-09-06%25282%2529.log/file

**Comment by 13xforever** [comment]
why mediafire lol, just zip it and attach here directly. you're trying to dump a damaged 2-layer disc, of course it will have problem on the first layer, and then on the second

**Comment by fac7orx** [comment]
[2019-09-06(2).log](https://github.com/13xforever/ps3-disc-dumper/files/3586818/2019-09-06.2.log)

The other log is too big to host. It incompassed a lot of crashes. Did realize mediafire is a bad thing I guess. I attached the smaller log.

I would agree with you with the disc is used and perhaps has some scratches. However, it also crashed on metal gear solid legacy collection extracting which is brand new and not scratched in any way.  So, I am a bit confused.

I managed to extract uncharted 1 and 2 ok, but the disc size was in the 20GB area vs ~40GB for the other discs.
